packages/lodestar/src/sync/regular/oneRangeAhead/processor.ts

```diff
@@ -47,7 +47,10 @@ export class BlockRangeProcessor implements IBlockRangeProcessor {
         resolve,
         signal,
       };
-      sortedBlocks.forEach((block) => this.chain.receiveBlock(block));
+      for (const block of sortedBlocks) {
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        this.chain.receiveBlock(block);
```

Comment:
we can use `await` here and remove the `eslint-disable-next-line` comment

Reply:
Hi, new to this repo but confirming you don't want to do something like this? I think what @mpetrunic mentioned:
```
Promise.all(sortedBlocks.map(block => this.chain.receiveBlock(block)));
```

Reply:
we really want to send blocks to processor queue in order, that's why I intentionally want to do that.
I think initially `Promise.all` was my first try but it's not a guarantee to execute them sequentially, at least in the spec.

Reply:
Makes sense!

Reply:
we can do something like this, if you feel comfortable with it:
```
public async processUntilComplete(blocks: SignedBeaconBlock[], signal: AbortSignal): Promise<void> {
    if (!blocks || !blocks.length) return;
    const sortedBlocks = sortBlocks(blocks);
    await Promise.all([
      new Promise((resolve) => {
        this.logger.info("Imported blocks for slots", {blocks: blocks.map((block) => block.message.slot)});
        const lastRoot = this.config.types.BeaconBlock.hashTreeRoot(sortedBlocks[sortedBlocks.length - 1].message);
        this.target = {
          root: lastRoot,
          resolve,
          signal,
        };
      }),
      (async function (chain) {
        for (const block of sortedBlocks) {
          await chain.receiveBlock(block);
        }
      })(this.chain)
    ]
    ) // end Promise.all
  }
```